sections/all_map_locations.tex

```diff
@@ -310,8 +311,8 @@ \subsection*{Other fields}
 \end{figure}
 \vfill
 
-\begin{figure*}[!hb]
-  \centering
-  \includegraphics[width=\linewidth]{\art/gold_dragon.jpg}
-\end{figure*}
+\begin{scaledfigure}[blanker]
+	\centering
+	\includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/gold_dragon.jpg}
+\end{scaledfigure}
```

Comment:
It seems that `scaledfigure` does not respect `\vfill`s surrounding them. The image is no longer centred in the remaining space, but glued to the text with a big blank space below it. Can you fix that?

Reply:
I have no idea how to do that sadly. I fiddled a lot to make it look like this.

Reply:
I wanted to use this library before, but I got scared away, once I noticed its manual is 600-pages long. The answer must be there somewhere :D

Reply:
As a last resort use a regular vspace.

Reply:
You know what, this is the only image with this problem. Since this is a final page with little text, in a section that's unlikely to change, maybe restore the old figure? Then it's ✅ from me. Everywhere else, `scaledfigure` looks good.